### PR TITLE
Changes 'episode number' field to be a title type.

### PR DESCRIPTION
**Pull request by afred**
Also, runs migrations if needed in AMS.reset_data!

### CODE REVIEW

**Comment by afred** [comment]
@adeelahmad this is currently failing for me locally, but it seems to be a web driver issue, so trying it on Travis to see what happens.

**Comment by afred** [comment]
Hmm.. ok, looks like it's not just a web driver issue.

File: Gemfile [hunk 1]
```diff
@@ -49,6 +49,8 @@ group :development, :test do
   gem 'bixby' # bixby == the hydra community's rubocop rules
   gem 'selenium-webdriver'
   gem 'chromedriver-helper'
+  # Alternative web driver to selenium; requires phantomjs
+  gem 'poltergeist'
```
Comment (adeelahmad) [inline Gemfile]: @afred can we get rid of this, if we are not using this anymore ?

Comment (afred) [reply Gemfile]: yep, sorry, stand by

**Pull request merged**